antora-playbook.yml

```diff
@@ -156,6 +156,7 @@ asciidoc:
     idprefix: '@'
     idseparator: '-@'
     tabs: tabs
+    tabs-sync-option: true
```

Comment:
The correct way to turn on an attribute (in this case global option) is an empty string, not the value "true". In other words:

```
tabs-sync-option: ''
```

Reply:
Ah, I've seen that, but as an empty string "looks" falsey, and it looks like any defined value counts as truthy, this felt more explicit?

Is the fact that `true` works undefined / likely to change in future?

Reply:
It may look weird, but it is how AsciiDoc has always been. The reason for it is that in a document, we can't rely on types like `true` and `false`. So it's better to be consistent with what is available there. It's the equivalent to:

```
:tabs-sync-option:
```

In the AsciiDoc Language project, we may eventually define a `true` value as a way to officially set and attribute (to an empty value)...but we haven't crossed that bridge. However, we can be certain that an empty value will always set the attribute.